docs/source/index.rst

```diff
@@ -17,6 +17,7 @@ Table of contents
        gpkit101
        installation
        gettingstarted
+       debugging

```

Comment:
I think either set up makes sense. @bqpd chose this order so he might have some thoughts